operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -516,6 +520,33 @@ type ScannerComponentSpec struct {
 	Monitoring *Monitoring `json:"monitoring,omitempty"`
 }
 
+// ScannerV4ComponentSpec defines settings for the central "scannerV4" component.
+type ScannerV4ComponentSpec struct {
+	// If you want to deploy the Red Hat Advanced Cluster Security ScannerV4, you can enable it here
+	// By default, the ScannerV4 is disabled.
+	//+kubebuilder:default=Disabled
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=1,displayName="Scanner V4 Component"
+	ScannerComponent *ScannerComponentPolicy `json:"scannerComponent,omitempty"`
```

Comment:
It is, and I also wondered why. The explanation I got on slack is that this is considered good practice in operator development since it enables us to add other values e.g "autosense" later on if we need to. So I just copied this from scanner v2 to keep it similar.